test/styleguide/legacy_bridge.js

```diff
@@ -17,14 +18,14 @@ window.Signal.Migrations = {
   getPlaceholderMigrations: () => {},
 };
 
-window.Signal.React = window.Signal.React = {};
+window.Signal.React = {};
```

Comment:
❤️

Reply:
Reminds me - perhaps this should be `window.Signal.Components`?